### PR TITLE
1 core concepts and styling

### PR DESCRIPTION
**Pull request by carlosverduzco**
I use yarn as the package manager of my project.

### CODE REVIEW

File: src/components/Navbar/Navbar.component.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import './Navbar.styles.css';
+
+function Navbar() {
+  return <></>;
```
Comment (marrossa86) [inline src/components/Navbar/Navbar.component.jsx:6]: **Question**: What is the idea of this component?

File: src/components/VideoPreviewCard/VideoPreviewCard.component.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import './VideoPreviewCard.styles.css';
+
+function VideoPreviewCard({ thumbnailURL, title, description, videoID }) {
+  if (!videoID) return <></>;
+  return (
+    <article className="VideoPreviewCard">
+      <a
+        className="VideoPreviewCard-URL"
+        href={`https://www.youtube.com/watch?v=${videoID}`}
+        target="_blanck"
```
Comment (marrossa86) [inline src/components/VideoPreviewCard/VideoPreviewCard.component.jsx]:
```suggestion
        target="_blank"
```

File: src/pages/Home/Home.page.jsx [hunk 3]
```diff
@@ -1,37 +1,43 @@
...
+  useEffect(() => {
+    fetch(
+      'https://gist.githubusercontent.com/jparciga/1d4dd34fb06ba74237f8966e2e777ff5/raw/f3af25f1505deb67e2cc9ee625a633f24d8983ff/youtube-videos-mock.json'
+    )
+      .then((res) => res.json())
+      .then(
+        (result) => {
+          setIsLoaded(true);
+          setItems(result.items);
+          console.log(result.items);
```
Comment (marrossa86) [inline src/pages/Home/Home.page.jsx]:
```suggestion
```

**Review by marrossa86 — COMMENTED** [review]
Hi Carlos,

This looks really great. I just left some minor comments. I think you are missing some elements in the navbar, but this PR looks really great. Thanks.

File: src/components/Navbar/Navbar.component.jsx (hunk 1 above)
Comment (carlosverduzco) [reply src/components/Navbar/Navbar.component.jsx:6]: My idea is make something like the navbar of youtube with a hamburguer menu on the left side and when its a mobile device on the bottom; in the hamburguer menu im thinking put the access to favorite section like in the example of the final app.

**Pull request merged**